stylix/target.nix

```
@@ -34,28 +34,83 @@
   config.lib.stylix =
     let
       cfg = config.stylix;
+      self = config.lib.stylix;
+
+      # Will wrap with (parentheses) if the expr contains operators with lower precedence than `&&`
+      wrapExprWith =
+        {
+          autoWrapExpr ? true,
+          trimExpr ? true,
+        }:
+        expr:
+        let
+          trim = if trimExpr then lib.strings.trim else lib.id;
+          trimmed = trim expr;
```

This might be simpler (and negligibly more performant) with:

```suggestion
          trimmed = if trimExpr then lib.strings.trim expr else expr;
```